src/Tools/dotnet-dsrouter/dotnet-dsrouter.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
I guess this needs to be insync with rest of diagnostic repro, the plan was to bump, but not sure that has happened yet over the board.

Reply:
The issue is that I don't know how to write a netcoreapp3.1-style ASP.NET server :grin: the documentation guides on docs.microsoft.com all got updated to net6+ and unlike the API reference there's not a way to see the older versions (AFAICT).

From talking with Tom it sounds like there's a plan to update the rest of the diagnostic tools to net6.0 at some point before netcoreapp3.1 is out of support.

But another way to think about it, is that dotnet-dsrouter was introduced to support EventPipe for ios and Android which happened in net6, so arguably this particular component has always been a net6.0 tool.

Reply:
@tommcdon, @hoyosjs Would there be any issues if we put dotnet-dsrouter and Microsoft.Diagnostics.WebSocketServer.csproj as part of this PR? dotnet-dsrouter is part of the global install of tools so not sure what would happen if we do.